test/wsdl-load-from-memory-test.js

```diff
@@ -39,6 +39,26 @@ describe('wsdl-load-from-memory-tests', function() {
         done()
       })
     });
+
+    it('should load a wsdl with no imports directly from memory and in a sync manner', function (done) {
+
+      var options = {
+        WSDL_CACHE: {},
+        forceSyncLoad: true
+      }
+      // Create the initial wsdl directly
+      var stockQuoteWsdl = new WSDL(stockQuoteWsdlContents, undefined, options)
+
+      // Load the wsdl fully once its been created in memory in a sync manner
+      var checkwsdl
```

Comment:
nitpick: please have `;` at the end of your statements.

Reply:
will do